package.json

```diff
@@ -38,16 +38,19 @@
     "@polkadot/ts": "^0.4.16",
     "@polkadot/x-bundle": "^8.1.2",
     "@types/jest": "^27.0.3",
+    "babel-loader": "^8.2.3",
     "babel-plugin-transform-vue-template": "^0.4.2",
     "empty": "^0.10.1",
+    "process": "^0.11.10",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-is": "^17.0.2",
     "react-native": "^0.66.3",
+    "stream-browserify": "^3.0.0",
```

Comment:
For both stream and buffer, actually need to track which culprits bring them in. (Really should be able to operate without)

Not for this PR, just a note.

Reply:
Buffer looks like some ledger lib

![image](https://user-images.githubusercontent.com/666169/145450592-12cf074d-5612-40fd-aa22-352ca1435b5a.png)

Reply:
And stream here:

![image](https://user-images.githubusercontent.com/666169/145450758-fec49ad4-74ce-4147-8913-f78423010972.png)

Reply:
```
ERROR in ../../node_modules/cipher-base/index.js 2:16-43
Module not found: Error: Can't resolve 'stream' in '/Users/yuri/Sites/polkadot/ui/node_modules/cipher-base'
```